File: API/HealthCareService/HealthCareService/Models/Domain/Patient.cs
using System;
using System.ComponentModel.DataAnnotations;

namespace HealthCareService.Models.Domain
{
    public class Patient
    {
        [Key]
        public string Id { get; set; }
        public string patient_name { get; set; }
        public string patient_email { get; set; }
        public string patient_gender { get; set; }
        // public string password{get;set;}
        public string patient_dob { get; set; }
        public DateTime registeredDate { get; set; }
        public long patient_mobile { get; set; }
        // public string location{get;set;}
    }
}

File: UI/HealthCareService/src/app/models/patient.ts
export class Patient {
    userId!: string;
    patient_name!: string;
    patient_gender!: string;
    patient_dob!: string;
    patient_mobile!: number;
    patient_email!: string;
    desc?: string;
    // regTime: Date;

    constructor(values: Object = {}) {
        Object.assign(this, values);
    }
}


// export class Patient {
//     // id: number;
//     firstName!: string;
//     lastName!: string;
//     gender!: string;
//     dob!: string;
//     mobile!: number;
//     email!: string;
//     description?: string;
//     registeredTime!: Date;

//     constructor(values: Object = {}) {
//         Object.assign(this, values);
//     }
// }